app/models/article_document.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+# This class is responsible for getting relevant
+# metadata from the Summon::Documents
+class ArticleDocument < Document
+  delegate :abstract, to: :document
+  delegate :end_page, to: :document
+  delegate :issue, to: :document
+  delegate :publication_title, to: :document
+  delegate :publisher, to: :document
+  delegate :start_page, to: :document
+  delegate :volume, to: :document
+
+  def title
+    full_sanitizer = Rails::HTML5::FullSanitizer.new
```

Comment:
This is nice!  Is the sanitization covered by any tests?

Reply:
You're right! Added!